src/clj/harja/palvelin/raportointi/raportit/ymparisto.clj

```diff
@@ -524,7 +524,11 @@ reittitiedot ja kokonaismateriaalimäärät raportoidaan eri tavalla."])
                           {:urakka_idt kontekstin-urakka-idt
                            :hoitokauden_alkuvuosi hoitokauden-alkuvuosi})
         talvisuolan-maxmaarat (group-by :urakka (map konv/alaviiva->rakenne talvisuolarajat))
-        talvisuolaa-suunniteltu-yhteensa (apply + (map :talvisuolaraja talvisuolarajat))
+        talvisuolaa-suunniteltu-yhteensa (reduce (fn [yht rivi]
```

Comment:
Samaan haluttuun lopputulokseen pääsisi myös vaihtamalla map-funktio keep:ksi
```clojure
(apply + (keep :talvisuolaraja talvisuolarajat))
```

Reply:
Erittäin hyvä idea. Vaihdoin